test/test_meta.py

```diff
@@ -1646,6 +1646,20 @@ def test_embedding_bag_byte_unpack(self):
             self.assertEqual(res.dtype, torch.float32)
             self.assertEqual(res.untyped_storage().data_ptr(), 0)
 
+    def test_segment_reduce_backward(self):
```

Comment:
Don't we have opinfo based test for these. This will give us much better coverage for inputs

Reply:
I saw an opinfo for the forward, but not the backward.

Lmk if you agree - it seems like we could either:
(1) Add a dedicated OpInfo for the backward op
(2) Add some generic testing in test_meta.py that loops over every existing OpInfo with meta tensors and invokes backward

(2) would be nice, but I'm not sure I'll have time for it. Would you prefer I add a new opinfo over the one-off test? (totally happy to, agreed the one-off test isn't great)

Reply:
I think a new opinfo is the right way to go tbh. We want all other tests on native op to run on the backward!